_includes/docs/user-guide/rule-engine-2-0/ce-enrichment-nodes.md

```diff
@@ -214,15 +214,15 @@ The incoming message from "WaterPressureSensor" will be as follows:
 msg: {"pressure": 75.5}, metadata: {"ts": "1616510425200"}
 ```
 
-The outbound message will be routed via **Success** chain and will include the following attribute and latest telemetry in the message:
+The outgoing message will be routed via **Success** chain and will include the following attribute and latest telemetry in the message:
 
 ```bash
 msg: {"pressure": 75.5, "ss_pressureThreshold": "80", "temperature": "{"ts":1718611002573, "value":23}"}, metadata: {"ts": "1616510425200"}
 ```
 
 ## originator attributes
 
-Adds message originator [attributes](/docs/user-guide/attributes/) and/or [latest telemetry](/docs/user-guide/telemetry/) values into the message or the message metadata.
+Enriches the outgoing message with the message originator's [attributes](/docs/user-guide/attributes/) or [latest telemetry](/docs/user-guide/telemetry/).
```

Comment:
It's still and/or. This node can enrich the message with attributes and the latest telemetry at the same time.